content/en/logs/guide/control-sensitive-logs-data.md

```diff
@@ -16,100 +16,94 @@ further_reading:
 
 ---
 
-Because your logs may contain sensitive data, you should be extra careful in how you handle them. Once you realize that you are ingesting sensitive data into Datadog, consider these two things:
+Logs may contain sensitive data, and should be handled carefully. If you are ingesting sensitive data into Datadog, consider the following:
 
-- If you have intentionally set up your logs to have sensitive data for legitimate troubleshooting and auditing purposes, use Roles Based Access Control to ensure that you have set up appropriate restrictions to ensure that only entitled users who have access to your Datadog account can access this data. For more information, see the [Logs Roles Based Access Control (RBAC) User's Guide][1] and learn how to configure it for your organization.
-- Address any unintentional logging of sensitive data to preemptively address any concerns down the road. Continue with this guide to learn about addressing that concern.
+- If you have intentionally set up your logs to have sensitive data for legitimate troubleshooting and auditing purposes, use the **Roles Based Access Control** to ensure that you have set up appropriate restrictions so only entitled users who have access to your Datadog account can access this data. For more information, see the [Logs Roles Based Access Control (RBAC) User's Guide][1] to learn how to configure it for your organization.
```

Comment:
I don't know why but "entitled" bristles me. 😂  Could we change it to "authorized" maybe?

Reply:
Definitely, this one slipped by! Will update.